jha/src/store/modules/enrolment-module.js

```diff
@@ -10,6 +10,15 @@ export const SET_CAPTCHA_TOKEN = 'setCaptchaToken';
 export const SET_SUBMISSION_DATE = 'setSubmissionDate';
 export const SET_REFERENCE_NUMBER = 'setReferenceNumber';
 export const SET_IS_INFO_COLLECTION_NOTICE_OPEN = 'setIsInfoCollectionNoticeOpen';
+// Eligibility Questionnaires
+export const SET_LIVE_IN_BC = 'setLiveInBC';
+export const SET_AWAY_OVER_30 = 'setAwayOver30';
+export const SET_STUDENT_MINOR_REFUGEE = 'setStudentMinorRefugee';
+export const SET_HAS_DOCUMENTS = 'setHasDocuments';
+export const SET_MEETS_FPC_CRITERIA = 'setMeetsFPCCriteria';
+export const SET_HAS_FPC_INFO = 'setHasFPCInfo';
+export const SET_MEETS_SB_CRITERIA = 'setMeetsSBCriteria';
+export const SET_HAS_SB_INFO = 'setHasSBInfo';
```

Comment:
Can you prefix these with "EQ" (eligibility questionnaire)? It makes it easier to identify the usage. For example:
```javascript
export const SET_EQ_LIVE_IN_BC = 'setEQLiveInBC';
export const SET_EQ_AWAY_OVER_30 = 'setEQAwayOver30';
```
Same with the state variables in this module.

Reply:
Changed variable names, also updated names to move identifier for which eligibility page the variable goes with.